jumpstart-foundation-models/text2text-generation-flan-t5-xl/text2text-generation-flan-t5-xl.ipynb

```diff
@@ -0,0 +1,1455 @@
+{
```

Comment:
<p>In this demo notebook, we demonstrate how to use the SageMaker Python SDK for deploying <strong>Foundation Models </strong>as an endpoint and use them for various NLP tasks<strong>. </strong>The Foundation models perform Text2Text Generation. It takes a prompting text as an input, and returns the text generated by the model according to the prompt. </p><p> </p><p>Here, we show how to use state-of-the-art pre-trained <strong><em>FLAN</em></strong> <strong>T5 models</strong> from <a href="https://huggingface.co/docs/transformers/model_doc/flan-t5" target="_blank">Hugging Face</a> for Text2Text Generation. One can directly use FLAN-T5 model for many NLP tasks, without finetuning the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3795/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
resolved.